src/components/Admin/Authentication/SignUp.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Text, Heading, Link, Divider} from '@chakra-ui/react';
+import {useAuth} from '../../../contexts/AuthContext';
+import {SubmitButton} from './Util';
+
+/**
+ * Props for SignUp component. Used for type safety.
+ */
+interface SignUpProps {
```

Comment:
Would it be better style/beneficial for the SignUpProps and SignInProps to be the same interface since they are identical?

Reply:
Good point, I'll change this